cms/djangoapps/contentstore/views/tests/test_certificates.py

```diff
@@ -320,7 +320,7 @@ def test_certificate_info_not_in_response(self):
 
         # in html response
         result = self.client.get_html(self._url())
-        self.assertNotIn('Test certificate', result.content)
+        self.assertNotIn('Test certificate', result.content.decode('utf-8'))
```

Comment:
Nit: Similar to above, see [FAQ #5](https://openedx.atlassian.net/wiki/spaces/AT/pages/998998951/Python+3+Conversion+and+Usage+Tips), which specifies preference for using `self.assertNotContains` instead of explicit decoding.